addons/stock_account/migrations/13.0.1.1/post-migration.py

```diff
@@ -1,329 +1,221 @@
-# Copyright 2020 ForgeFlow <http://www.forgeflow.com>
-# Copyright 2020 Andrii Skrypka
+# Copyright 2021 Tecnativa - Carlos Dauden
+# Copyright 2021 Tecnativa - Sergio Teruel
```

Comment:
why?

Reply:
Sorry, I created a new file with my default template...
Feel free to change what you want.